apps/calendar/js/views/event_base.js

```diff
@@ -200,21 +158,17 @@ EventBase.prototype = {
 
     classList.add(this.LOADING);
 
-    dayObserver.findAssociated(id).then(record => {
+    bridge.fetchRecord(id).then(record => {
```

Comment:
I kept as much from old logic as possible.. since it needs to handle the error it wouldn't be too different (would need to wrap into `try/catch`..)